native_implemented/otp/src/erlang/are_equal_after_conversion_2/test/with_function_left.rs

```diff
@@ -80,14 +90,24 @@ fn with_different_native_right_returns_false() {
                         Term::NONE
                     };
                     let left_term = arc_process
-                        .export_closure(module, function, arity, Some(left_native as _))
+                        .export_closure(
+                            module,
+                            function,
+                            arity,
+                            Some(NonNull::new(left_native as _).unwrap()),
+                        )
                         .unwrap();
 
                     extern "C" fn right_native() -> Term {
                         Term::NONE
                     };
                     let right_term = arc_process
-                        .export_closure(module, function, arity, Some(right_native as _))
+                        .export_closure(
+                            module,
+                            function,
+                            arity,
+                            Some(NonNull::new(right_native as _).unwrap()),
```

Comment:
`NonNull::new` returns `Option<NonNull<T>>`, so the `Some(NonNull::new().unwrap()))` is redundant

Reply:
I saw it did that, but It was more that I wanted to be explicit that it was non-null and wrap that with Some.  I get from the code I wrote that will always return `Some(NonNull)`, but I don't want it to silently accept `None`.

Reply:
I'll change it anyway.  Any reduction in the amount of test code is probably for the best.